corehq/apps/dashboard/templates/dashboard/dashboard_new_user.html

```diff
@@ -12,83 +12,33 @@
 {% block dashboard-content %}
 {% initial_page_data 'templates' templates %}
 <div class="page-header">
-    {% if request|toggle_enabled:"APP_MANAGER_V2" %}
-        <h1>{% trans 'Would you like to work on an existing app or create a new one?' %}</h1>
-    {% else %}
-        <h1>{% trans 'What type of app would you like to build?' %}</h1>
-    {% endif %}
+    <h1>{% trans 'What type of app would you like to build?' %}</h1>
 </div>
 <div class="row">
-    {% if request|toggle_enabled:"APP_MANAGER_V2" %}
```

Comment:
This diff came out weird; all I'm doing here is removing the v2 part of the `if` block.